### ISSUE TITLE
Turtle fails with internal error when not using named turtle

### ISSUE THREAD

**Issue by deansheather** [skulpt/skulpt#697]
Using the following turtle code yields an internal error. Tested using the `simpleskulpt.html` file on the project homepage. Naming the turtle (using the default code in `simpleskulpt.html`) doesn't seem to produce this problem. I copied the error from Firefox console, I apologize for the formatting.

This code previously worked on Trinket.io yesterday.

```py
from turtle import *
forward(100)

print "Hello World" 

```

```
wrapped method failed  skulpt.min.js%20line%20994%20%3E%20eval:2082:21

wrapperFn@http://www.skulpt.org/static/skulpt.min.js line 994 > eval:2079:17
Sk.builtin.func.prototype.tp$call@http://www.skulpt.org/static/skulpt.min.js:155:366
Sk.misceval.applyOrSuspend@http://www.skulpt.org/static/skulpt.min.js:272:391
Sk.misceval.callsimOrSuspend@http://www.skulpt.org/static/skulpt.min.js:263:93
$scope0@http://www.skulpt.org/static/skulpt.min.js line 994 > eval:20:104
@http://www.skulpt.org/static/skulpt.min.js line 994 > eval:39:1
x@http://www.skulpt.org/static/skulpt.min.js:994:373
Sk.importModuleInternal_@http://www.skulpt.org/static/skulpt.min.js:993:346
Sk.importMainWithBody@http://www.skulpt.org/static/skulpt.min.js:996:358
runit/myPromise<@file:///home/dean/Downloads/simpleskulpt.html:37:15
Sk.misceval.asyncToPromise/<@http://www.skulpt.org/static/skulpt.min.js:265:41
Sk.misceval.asyncToPromise@http://www.skulpt.org/static/skulpt.min.js:264:49
runit@file:///home/dean/Downloads/simpleskulpt.html:36:20
onclick@file:///home/dean/Downloads/simpleskulpt.html:1:1
  skulpt.min.js%20line%20994%20%3E%20eval:2083:21

ExternalError: TypeError: instance[method] is undefined on line 2  simpleskulpt.html:43:8

```

**Comment by eah13** [comment]
We deployed a whole bunch of upstream code this morning and obviously did not catch this in testing. Thanks a ton for reporting! Working on a roll back of our skulpt version and will dig in tomorrow and report here.  Unfortunately I don't have a good suggestion for anyone else of where to roll back to (unless you wan to use version 0.11.1.13 of our fork). See trinketapp/skulpt#28 for the long list of PRs that are candidates for causing this bug.

**Comment by eah13** [comment]
I've got to bounce off to a few other things until later today but for anyone else investigating this it seems to traces to the `args.slice()` call here:

```js
        /**
        * The constructor is a stub, that gets called from object.__new__
        * @constructor
        */
        klass = function (args, kws) {
            var args_copy;

            // Call up through the chain in case there's a built-in object
            // whose constructor we need to initialise
            if (klass.prototype.tp$base !== undefined) {
                if (klass.prototype.tp$base.sk$klass) {
                    klass.prototype.tp$base.call(this, args, kws);
                } else {
                    // Call super constructor if subclass of a builtin
                    args_copy = args.slice();
                    args_copy.unshift(klass, this);
                    Sk.abstr.superConstructor.apply(undefined, args_copy);
                }
            }

            this["$d"] = new Sk.builtin.dict([]);
            this["$d"].mp$ass_subscript(new Sk.builtin.str("__dict__"), this["$d"]);
        };
```

Once that's fixed there's more, but this is a good place to start.

That's from #694 I think.

**Comment by albertjan** [comment]
This is fixed by #698
